### PR TITLE
misc cleanup

### PR DESCRIPTION
**Pull request by jaycenhorton**

### CODE REVIEW

File: contracts/IMarket.sol [hunk 1]
```diff
@@ -2,17 +2,16 @@
...
-  function getRestrictedNoriAddress() external view returns (address);
-
   /**
    * @notice Releases a removal from the market.
    * @dev This function is called by the Removal contract when releasing removals.
-   *
    * @param removalId The ID of the removal to release.
    */
   function release(uint256 removalId) external;
+
```
Comment (jaycenhorton) [inline contracts/IMarket.sol:11]: reordered according to the solium rule

File: contracts/IRestrictedNORI.sol [hunk 2]
```diff
@@ -24,4 +16,11 @@ interface IRestrictedNORI {
     uint8 methodology,
     uint8 methodologyVersion
   ) external;
+
+  /**
+   * @notice Check the existence of a schedule.
+   * @param scheduleId The token ID of the schedule for which to check existence.
+   * @return Returns a boolean indicating whether or not the schedule exists.
+   */
+  function scheduleExists(uint256 scheduleId) external view returns (bool);
```
Comment (jaycenhorton) [inline contracts/IRestrictedNORI.sol:25]: reordered according to the solium rule

File: contracts/Market.sol [hunk 3]
```diff
@@ -249,7 +249,8 @@ contract Market is
    * @notice Emitted when the ERC20 token that would be transferred to the RestrictedNORI contract is not the token
    * address that RestrictedNORI was configured to wrap.
    * @param amount The amount of RestrictedNORI in the transfer attempt.
-   * @param currentHoldbackPercentage The holdback percentage for this removal id's project at the time of this event emission.
+   * @param currentHoldbackPercentage The holdback percentage for this removal id's project at the time of this event
```
Comment (jaycenhorton) [inline contracts/Market.sol:252]: conforms to the solium max line length rule

File: contracts/Market.sol [hunk 4]
```diff
@@ -678,24 +675,22 @@ contract Market is
...
-   * tokens for an ERC721 certificate token and transfers ownership of removal tokens supplied only from the specified
-   * supplier to that certificate, without charging a transaction fee. If the specified supplier does not have enough
-   * carbon removals for sale to fulfill the order the transaction will revert.
-   * @dev See [here](https://docs.openzeppelin.com/contracts/4.x/api/token/erc20#IERC20-approve-address-uint256-) for more.
-   * The message sender must have granted approval to this contract to authorize this market to transfer the sender's
-   * supported ERC20 to complete the purchase. A certificate is issued by the Certificate contract
-   * contract to the specified recipient and the ERC20 is distributed to the supplier of the carbon removal and potentially
-   * to the RestrictedNORI contract that controls any restricted portion of the ERC20 owed to the supplier.
-   *
+   * @notice An overloaded version of `swap` that additionally accepts a supplier address and will exchange supported
```
Comment (jaycenhorton) [inline contracts/Market.sol:678]: conforms to max line length rule

File: contracts/Removal.sol [hunk 5]
```diff
@@ -694,6 +670,27 @@ contract Removal is
     return super.supportsInterface({interfaceId: interfaceId});
   }
 
+  /**
```
Comment (jaycenhorton) [inline contracts/Removal.sol:673]: reordered according to the solium rule

File: contracts/RestrictedNORI.sol [hunk 6]
```diff
@@ -534,7 +534,6 @@ contract RestrictedNORI is
         scheduleTokenId: scheduleId,
         balance: balanceOf({account: account, id: scheduleId}),
         claimableAmount: schedule.claimableBalanceForScheduleForAccount({
-          scheduleId: scheduleId,
```
Comment (jaycenhorton) [inline contracts/RestrictedNORI.sol:537]: unused param that was removed during the last audit

File: contracts/RestrictedNORI.sol [hunk 7]
```diff
@@ -694,38 +692,6 @@ contract RestrictedNORI is
     ] = durationInSeconds;
   }
 
-  /**
-   * @notice Token transfers disabled.
-   * @dev Transfer is disabled because keeping track of claimable amounts as tokens are
-   * claimed and transferred requires more bookkeeping infrastructure that we don't currently
-   * have time to write but may implement in the future.
-   */
-  function safeTransferFrom(
```
Comment (jaycenhorton) [inline contracts/RestrictedNORI.sol:703]: reordered according to the solium rule

File: contracts/RestrictedNORILib.sol [hunk 8]
```diff
@@ -129,15 +129,13 @@ library RestrictedNORILib {
    * using totals constructed from current balances and claimed amounts, and then subtract anything that
    * account has already claimed.
    * @param schedule The schedule to calculate the claimable amount for.
-   * @param scheduleId The schedule ID to calculate the claimable amount for.
```
Comment (jaycenhorton) [inline contracts/RestrictedNORILib.sol:132]: unused param removed during prior audit

**Review by amiecorso — APPROVED** [review]
nice thanks for this

**Pull request merged**